code/modules/power/cell.dm

```diff
@@ -280,6 +280,7 @@
 	maxcharge = 50000
 	rating = 12
 	ratingdesc = FALSE
+	self_recharge = 1

```

Comment:
That only halves maxcharge, they still won't use power.
An infinite cell with no power will still work anyway, as again, it's ```use()``` proc doesn't even check if it has the charge necessary to work, it just -works-